ibex_configs.yaml

```diff
@@ -40,22 +40,35 @@ experimental-maxperf:
 experimental-maxperf-pmp:
   RV32E                    : 0
   RV32M                    : 1
-  RV32B                    : 0
+  RV32B                    : "ibex_pkg::RV32BNone"
   BranchTargetALU          : 1
   WritebackStage           : 1
   MultiplierImplementation : "single-cycle"
   PMPEnable                : 1
   PMPGranularity           : 0
   PMPNumRegions            : 16
 
-# experimental-maxperf-pmp config above with bitmanip extension
+# experimental-maxperf-pmp config above with balanced bitmanip extension
+experimental-maxperf-pmp-bmbalanced:
+  RV32E                    : 0
+  RV32M                    : 1
+  RV32B                    : "ibex_pkg::RV32BBalanced"
+  BranchTargetALU          : 1
+  WritebackStage           : 1
+  MultiplierImplementation : "single-cycle"
+  PMPEnable                : 1
+  PMPGranularity           : 0
+  PMPNumRegions            : 16
+
+# experimental-maxperf-pmp config above with full bitmanip extension
 experimental-maxperf-pmp-bm:
```

Comment:
This needs to be changed to -bmfull to match the default in the DV makefile.

Reply:
Oh, yes. Actually, `-bm` is the original name. I changed it to `-bmfull` because of the new balanced config. I will have to change it in `azure-pipelines.yml` as well, if that's ok.

Reply:
Great, thanks!